test/serializable_resource_test.rb

```diff
@@ -3,8 +3,8 @@
 module ActiveModel
   class SerializableResourceTest < Minitest::Test
     def setup
-      @resource = Profile.new({ name: 'Name 1', description: 'Description 1', comments: 'Comments 1' })
-      @serializer = ProfileSerializer.new(@resource)
+      @resource = Profile.new({ name: 'Name 1', description: 'Description 1', comments: 'Comments 1', options: 'one' })
+      @serializer = ProfileWithOptionsSerializer.new(@resource)
       @adapter = ActiveModel::Serializer::Adapter.create(@serializer)
       @serializable_resource = ActiveModel::SerializableResource.new(@resource)
```

Comment:
You are right about my wrong test. The test passed but the assertions are both wrong
```
[1] pry(#<ActiveModel::SerializableResourceTest>)> @adapter.serializable_hash(options)
=> {:name=>"Name 1", :description=>"Description 1", :options=>{}}
[2] pry(#<ActiveModel::SerializableResourceTest>)> @serializable_resource.serializable_hash(options)
=> {:name=>"Name 1", :description=>"Description 1", :options=>{}}
```

It is fixed on master, probably #1166 
```
@adapter.serializable_hash(options)
=> {:name=>"Name 1", :description=>"Description 1", :options=>"one"}
[2] pry(#<ActiveModel::SerializableResourceTest>)> @serializable_resource.serializable_hash(options)
=> {:name=>"Name 1", :description=>"Description 1", :options=>"one"}
```